src/Compiler/test/Microsoft.NET.Sdk.Razor.SourceGenerators.Tests/RazorSourceGeneratorTests.cs

```diff
@@ -3189,73 +3189,17 @@ public async Task IncrementalCompilation_NothingRuns_When_AdditionalFiles_HaveSa
             var compilation = await project.GetCompilationAsync();
             var (driver, additionalTexts) = await GetDriverWithAdditionalTextAsync(project);
 
-            var result = RunGenerator(compilation!, ref driver)
-                            .VerifyPageOutput(
-@"#pragma checksum ""Pages/Index.razor"" ""{ff1816ec-aa5e-4d10-87f7-6f4963833460}"" ""6b5db227a6aa2228c777b0771108b184b1fc5df3""
-// <auto-generated/>
-#pragma warning disable 1591
-namespace MyApp.Pages
-{
-    #line default
-    using global::System;
-    using global::System.Collections.Generic;
-    using global::System.Linq;
-    using global::System.Threading.Tasks;
-    using global::Microsoft.AspNetCore.Components;
-    #line default
-    #line hidden
-    #nullable restore
-    public partial class Index : global::Microsoft.AspNetCore.Components.ComponentBase
-    #nullable disable
-    {
-        #pragma warning disable 1998
-        protected override void BuildRenderTree(global::Microsoft.AspNetCore.Components.Rendering.RenderTreeBuilder __builder)
-        {
-            __builder.AddMarkupContent(0, ""<h1>Hello world</h1>"");
-        }
-        #pragma warning restore 1998
-    }
-}
-#pragma warning restore 1591
-",
-@"#pragma checksum ""Pages/Counter.razor"" ""{ff1816ec-aa5e-4d10-87f7-6f4963833460}"" ""0de17e526cd536d59072aa0e924e99111b16b97a""
-// <auto-generated/>
-#pragma warning disable 1591
-namespace MyApp.Pages
-{
-    #line default
-    using global::System;
-    using global::System.Collections.Generic;
-    using global::System.Linq;
-    using global::System.Threading.Tasks;
-    using global::Microsoft.AspNetCore.Components;
-    #line default
-    #line hidden
-    #nullable restore
-    public partial class Counter : global::Microsoft.AspNetCore.Components.ComponentBase
-    #nullable disable
-    {
-        #pragma warning disable 1998
-        protected override void BuildRenderTree(global::Microsoft.AspNetCore.Components.Rendering.RenderTreeBuilder __builder)
-        {
-            __builder.AddMarkupContent(0, ""<h1>Counter</h1>"");
-        }
-        #pragma warning restore 1998
-    }
-}
-#pragma warning restore 1591
-");
-
+            var result = RunGenerator(compilation!, ref driver);
             Assert.Empty(result.Diagnostics);
             Assert.Equal(2, result.GeneratedSources.Length);
 
             eventListener.Events.Clear();
 
-            result = RunGenerator(compilation!, ref driver)
-                        .VerifyOutputsMatch(result);
+            result = RunGenerator(compilation!, ref driver);
```

Comment:
Can we keep these `.VerifyOutputsMatch(result)`? They seem useful in asserting the output doesn't change.